src/gluonts/dataset/jsonl.py

```diff
@@ -22,14 +22,12 @@
 
 
 def load(file_obj):
-    for line in file_obj:
-        yield json.loads(line)
+    return map(json.loads, file_obj)
```

Comment:
Doesn't this change the behavior when we change the generator to this? Is `map` evaluated lazy?

Reply:
Yes, this is lazy. `map` and `filter` don't return lists anymore in Python 3 (they used to in Python 2).